cs/routing.texy

```diff
@@ -323,6 +323,27 @@ Pokud potřebujete do HTTPS překlopit celou aplikaci, použijte `Route::$defaul
 Route::$defaultFlags = Route::SECURED;
 \--
 
+.[caution]
+Route::$defaultFlags v Nette 2.4 již nepodporované
+
+Náhrada za `Route::$defaultFlags`
+---------------------------------
+
+Jestliže využíváme pouze HTTP nebo HTTPS, tak se již dále nemusíme o překlápění aplikace starat. Ovšem jestliže využíváme HTTP a HTTPS, a potřebovali bychom jinou aplikaci jako subdoménu, kterou ovšem potřebujeme celou překlopit do HTTPS, tak to můžeme provést jednoduchým zápisem do souboru v hlavním adresáři aplikaci .htaccess
```

Comment:
co například takhle: Jestliže využíváme pouze HTTP nebo HTTPS, tak se již dále nemusíme o překlápění aplikace starat. Ovšem jestliže využíváme HTTP a HTTPS, a potřebovali bychom celou aplikaci překlopit do HTTPS, tak to můžeme provést jednoduchým zápisem do souboru .htaccess v hlavním adresáři naší aplikace.